dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/dispatcher/internal/channel/HttpResponseImpl.java

```diff
@@ -144,6 +146,14 @@ public void setHeader(String name, String value) {
         this.message.setHeader(name, value);
     }
 
+    /*
+     * @see com.ibm.websphere.http.HttpResponse#setHeader(com.ibm.wsspi.http.channel.values.HttpHeaderKeys, java.lang.String)
```

Comment:
Comment is wrong.  Should referenceHttpResponseExt

Reply:
Good catch - done